examples/cfd/stokes_mgn/inference.py

```diff
@@ -45,37 +48,38 @@
         + "pip install pyvista"
     )
 
-C = Constants()
-
 
 class MGNRollout:
-    def __init__(self, wb):
+    def __init__(self, cfg: DictConfig, wb, logger):
```

Comment:
Note on `wb`: this is the `wandb` module alias, not an instance, so makes little sense to pass it all over the place as an argument.
I would suggest replacing all occurrences of `wb` with just `wandb` (similar to what I did in Ahmed body and vortex shedding samples).

Reply:
Thanks, resolved!